dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -280,11 +307,29 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         return wil::str_printf<std::wstring>(L"%ws%ws%ws", m_inlineImage.c_str(), m_heroImage.c_str(), m_appLogoOverride.c_str());
     }
 
-    std::wstring AppNotificationBuilder::GetButtons()
+    std::wstring AppNotificationBuilder::GetActions()
```

Comment:
We have an opportunity here to do some extra validation that goes beyond what the xml schema prescribes. For example, we couold ensure that a input-hint in a button elements, refers an existing texbox.

Reply:
Captured here: https://microsoft.visualstudio.com/OS/_workitems/edit/40746502